lib/common/socket/evloop/select.c.h

```diff
@@ -20,7 +20,9 @@
  * IN THE SOFTWARE.
  */
 #include <stdio.h>
+#ifndef _WIN32
 #include <sys/select.h>
+#endif
```

Comment:
`sys/select.h` is part of POSIX (http://pubs.opengroup.org/onlinepubs/009696899/basedefs/sys/select.h.html) and therefore does not need to be surrounded by `#if`.